cloudvcl/settings.py

```diff
@@ -119,9 +119,9 @@
 
 STATIC_URL = '/static/'
 
-# Location on the hard drive where we want files uploaded to
-MEDIA_ROOT = os.path.join(BASE_DIR, 'media')
-MEDIA_URL = '/media/'
+# # Location on the hard drive where we want files uploaded to
```

Comment:
I removed it from my settings.py file. Yet for some reason it still shows the previous settings.py file